src/commands/nickname.ts

```diff
@@ -109,16 +109,16 @@ export class Nickname implements ApplicationCommand {
             const commandUser = command.guild?.members.cache.find(m => m.id === command.user.id)!;
             // We know that the user option is in every subcommand.
             const user = command.options.getUser("user", true);
-            const isTrusted = commandUser?.roles.cache.has(config.ids.trusted_role_id);
-            const isSameUser = user.id === commandUser.user.id;
+            const trusted = commandUser?.roles.cache.has(config.ids.trusted_role_id);
+            const sameuser = user.id === commandUser.user.id;
```

Comment:
Warum hast du das jetzt umbenannt? Find das ehrlich gesagt mit `is` Prefix besser. Kannst auch die Funktion [isTrusted](https://github.com/NullDev/CSZ-Bot/blob/267b668a936b3424c882aee19cb0253a6d9ac3e9/src/utils/userUtils.ts#L36) aus den Utils nutzen